src/components/Checkbox.js

```diff
@@ -67,7 +71,7 @@ function Checkbox(props) {
     };
 
     return (
-        <Pressable
+        <PressableWithFeedback
             disabled={props.disabled}
```

Comment:
👋 Just a heads-up that this has caused a small regression in https://github.com/Expensify/App/issues/22413
`BaseGenericPressable` didn't have the logic to disable `onKeyDown` handler when `disabled` prop is true
We resolved this by adding the following condition
```js
onKeyDown={!isDisabled ? onKeyDown : undefined}